backend/tests/middlewares/validationMiddleware.test.js

```
@@ -0,0 +1,88 @@
+const { validateRequestBody } = require('../../src/middlewares/validationMiddleware');
+const Joi = require('joi');
+
+describe('validateRequestBody', () => {
+  let mockReq, mockRes, mockNext;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)